release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -68,6 +68,12 @@ You can incorporate example Azure Resource Manager (ARM) templates provided by R
 ==== Increased size of Azure subnets within the machine CIDR
 The {product-title} installation program for Microsoft Azure now creates subnets as large as possible within the machine CIDR. This lets the cluster use a machine CIDR that is appropriately sized to accommodate the number of nodes in the cluster.
 
+[id="ocp-4-9-aws-china-regions"]
+==== Support for AWS regions in China
+{product-title} {product-version} introduces support for AWS regions in China. You can now install and update {product-title} clusters in the `cn-north-1` (Beijing) and `cn-northwest-1`(Ningxia) regions.
+
+//For more information, ../installing/installing_aws/installing-aws-china.adoc[Installing a cluster on AWS into a China region].
```

Comment:
I just wanted to make sure that I pointed out that this link would be commented out. I am guessing that's due to the page not being completed.

Reply:
Correct. The help topic is still in flight. I will make sure to add this back in a separate PR. However, wanted to get the actual content in for the Sept 1 release date.